resources/views/edit.blade.php

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Insert</title>
+    <!-- CSS only -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+</head>
+<body>
+    <h1 class="text-center">変更</h1>
+    <div class= "container" >
+        <form method= "POST" action="{{route('post_update',$posts->id )}}">
+            @csrf
+            <div class="mb-3">
+                <label for=""><b>タイトル</b></label>
+                <input type="text" name="title" class = "form-control" value="{{$posts->title}}">
```

Comment:
```suggestion
                <input type="text" name="title" class = "form-control" value="{{ $posts->title }}">
```
sửa tương tự bên dưới nha